include/gauxc/xc_integrator_settings.hpp

```diff
@@ -16,4 +16,9 @@ struct IntegratorSettingsSNLinK : public IntegratorSettingsEXX {
   double k_tol      = 1e-10;
 };
 
+struct IntegratorSettingsEXCVXC { virtual ~IntegratorSettingsEXCVXC() noexcept = default; };
```

Comment:
```suggestion
struct IntegratorSettingsXC { virtual ~IntegratorSettingsXC() noexcept = default; };
```
I can think of a few things we can put here that will propagate to a number of different places. Best to leave it a generic name.

Reply:
Agreed, made that change.